public/app/plugins/panel/graph/jquery.flot.events.ts

```diff
@@ -1,10 +1,15 @@
 import angular from 'angular';
 import $ from 'jquery';
 import _ from 'lodash';
+//@ts-ignore
 import Drop from 'tether-drop';
 
 /** @ngInject */
-export function createAnnotationToolip(element, event, plot) {
+export function createAnnotationToolip(
+  element: any[] | JQLite,
+  event: any,
+  plot: { getOptions: () => { events: { manager: any } } }
```

Comment:
this seems to be reused in two locations, would be nice with an interface type for it